systemtest/src/main/java/io/strimzi/systemtest/logs/LogCollector.java

```diff
@@ -143,9 +144,9 @@ public synchronized void collect() {
 
         // it's not test suite but test case, and we are gonna collect logs
         if (!this.collectorElement.getTestMethodName().isEmpty()) {
-            // fetch test suite extensionContext
-            // @ParallelSuite -> this is generated namespace
-            if (StUtils.isParallelSuite(extensionContext.getParent().get())) {
+            // @ParallelSuite -> this is generated namespace but we collect logs only iff STRIMZI_RBAC_SCOPE=CLUSTER
+            // because when e we run STRIMZI_RBAC_SCOPE=NAMESPACE mode we use one namespace (i.e., clusterOperatorNamespace).
```

Comment:
```suggestion
            // @ParallelSuite -> this is generated namespace but we collect logs only if STRIMZI_RBAC_SCOPE=CLUSTER
            // because when we run STRIMZI_RBAC_SCOPE=NAMESPACE mode we use one namespace (i.e., clusterOperatorNamespace).
```